l10n_it_pec_messages/model/mail_mail.py

```diff
@@ -77,3 +78,34 @@ def send_get_email_dict(self, cr, uid, mail, partner=None, context=None):
             email_to = [formataddr((partner.name, partner.pec_mail))]
             res['email_to'] = email_to
         return res
+
+    def send(
+        self, cr, uid, ids, auto_commit=False, recipient_ids=None, context=None
+    ):
+        for mail in self.browse(cr, uid, ids, context=context):
+            if mail.mail_server_id.pec and recipient_ids:
+                #remove duplicate id if there are
+                recipient_ids = sorted(set(recipient_ids))
+                if recipient_ids:
+                    address_lst = ''
+                    #save old value of email_to
+                    if mail.email_to:
+                        address_lst = mail.email_to
+                    partner_obj = self.pool.get('res.partner')
+                    existing_recipient_ids = partner_obj.exists(
+                        cr, SUPERUSER_ID, recipient_ids, context=context)
+                    for partner in partner_obj.browse(
+                        cr, SUPERUSER_ID, existing_recipient_ids,
+                        context=context
+                    ):
+                        address_lst += formataddr(
+                            (partner.name, partner.pec_mail)) + ' '
```

Comment:
no i've tried but not work and this separation is made later here
https://github.com/OCA/OCB/blob/8.0/openerp/addons/base/ir/ir_mail_server.py#L323